core/src/main/resources/org/apache/struts2/default.properties

```diff
@@ -219,4 +219,8 @@ struts.ognl.enableExpressionCache=true
 ### Indicates if Dispatcher should handle unexpected exceptions by calling sendError()
 ### or simply rethrow it as a ServletException to allow future processing by other frameworks like Spring Security
 struts.handle.exception=true
+
+### applies maximum length allowed on OGNL expressions for security enhancement
```

Comment:
The value `200` still seems a bit short, and likely to be exceeded by a fair number of legitimate expressions.
Maybe `256` characters ( 0.25 KB ) would be a little more flexible for a starting point (but still short enough to have some benefit) ?